README.md

```diff
@@ -0,0 +1,36 @@
+# Archipelago Protocol
+
+Archipelago is a protocol for creating permissionless marketplaces for buying
+and selling collections of NFTs that are priced in ETH.
```

Comment:
Is "buying and selling *collections* of NFTs" (emph. mine) confusing?
This doesn't refer to bundles, and you're not actually buying the whole
Art Blocks collection itself. Does "buying and selling NFTs that are
priced in ETH" suffice, or is there nuance here that I'm missing?

Reply:
Changed.